blottertrax/helper/title_parser.py

```diff
@@ -46,14 +46,14 @@ def create_parsed_submission_from_submission(submission) -> ParsedSubmission:
                     break
 
         if song_title is None:
-            raise LookupError
+            return ParsedSubmission(False, '', '', '', submission.url)
 
         # Remove everything between () and []
         song_title = re.sub(r"\((.*?)\)", "", song_title)
         song_title = re.sub(r"\[(.*?)\]", "", song_title)
         song_title = song_title.strip()
 
         if artist is not None:

```

Comment:
Shouldn't need this check or line 59 as the artist being None was checked above at L24 https://github.com/martijnboers/BlotterTrax/pull/45/files#diff-fe0076842fd5a3dcf6a1829ff40423a3R24